### ISSUE TITLE
My site is down with the latest version of Gantry 5.5.16

### ISSUE THREAD

**Issue by webmasterpaca** [gantry/gantry5#3138]
Hi,
My site is down with the latest version of Gantry 5.5.16
Message : Failed to load 'g5_helium' template: Please install Gantry 5 Framework!
Thanks

**Comment by J-and-F-and-S** [comment]
Same here, I had to re-install Gantry 5.5.15 on a couple of sites

**Comment by N8Solutions** [comment]
@webmasterpaca did you also update the helium template or only the framework?
If you updated the template as well as the framework I would suggest reinstalling the previous version. You can find the previous releases here:
https://github.com/gantry/gantry5/releases

@hexplor can you look chime in here?

**Comment by OldMan-Jakob** [comment]
Hi
Tried installing this version on three different sites. Error Messages in each case:
"Install path does not exist.
Package Update: There was an error installing an extension: plg_system_gantry5_v5.5.15.zip
danger
Error installing package"

Regards

**Comment by gphood** [comment]
@OldMan-Jakob - I've just downloaded the package and had a look at the manifest file and enabled="1" has been removed from the line for the plugin, which I assume is why we're getting the install error. Maybe they disabled it to fix the issue reported earlier? 

My colleague reported issues with our sites as well, which have templates based on Helium that I've tried to replicate, but can't because the update won't install. I can bypass that error but probably worth waiting for an update from the Gantry devs first.

**Comment by galvani66** [comment]
Habe soeben auch auf pkg_gantry5_v5.5.16 aktualisiert, die Webseite ist direkt mit Interner Serverfehler 500 ausgestiegen,
habe ein Backup eingespielt und werde das Update nun in einer Testumgebung testen.

**Comment by tonamphong** [comment]
Hi there,
I already reported here https://github.com/gantry/gantry5/issues/3136
1. Just download the CI build version https://gantry.org/downloads#ci-builds and install it to fix the issue
2. Or you can unzip joomla-pkg_gantry5_v5.5.16.zip -> open pkg_gantry5.xml then change version number 5.5.15 to 5.5.16 for all packages(line 17-31) -> zip the installation package and reinstall it.

Hexplor already fixed it, I think the new release coming soon.

**Comment by tonamphong** [comment]
@hexplor 
CI Build package working but the version number is not defined. Can you check it?
See
![ver1](https://user-images.githubusercontent.com/2299347/229735963-433ee403-7d4b-429d-b69b-a85a20841dc8.png)
![ver2](https://user-images.githubusercontent.com/2299347/229736016-50a6835e-003c-46ee-b8c9-c7e248bd4e54.png)

**Comment by jmgroud** [comment]
Error on the extension "plg_system_gantry5_v5.5.15.zip" during the installation of "Gantry 5.5.16".
Where does this message come from and why this version difference between Gantry and the plugin?

"Warning
The installation path does not exist
Update Pack: error while installing the extension plg_system_gantry5_v5.5.15.zip"

When I try to install Gantry in version 5.5.15, everything is OK!
Is it a bug? An operation to do on the server to remove this message? A folder to create ? etc...

I am under Joomla 4.2.9 and PHP 8.1.17
![20230404 - Err Inst Gantry 5 5 16](https://user-images.githubusercontent.com/1281543/229737572-a9b50254-df23-437e-800e-b8878a21f8f7.jpg)

**Comment by hexplor** [comment]
> @hexplor CI Build package working but the version number is not defined. Can you check it? See ![ver1](https://user-images.githubusercontent.com/2299347/229735963-433ee403-7d4b-429d-b69b-a85a20841dc8.png) ![ver2](https://user-images.githubusercontent.com/2299347/229736016-50a6835e-003c-46ee-b8c9-c7e248bd4e54.png)

Correct. Reverting takes a while, and it was showing tokens instead of correct version. It's fine now. Reverted to 5.5.15 and need to investigate what happened. I was replacing tokens around 2 AM so probably missed something :(

**Comment by hexplor** [comment]
Anyways, does the release files here https://github.com/gantry/gantry5/releases/tag/5.5.16 works fine for you or these are also broken?

**Comment by tonamphong** [comment]
It's broken bro, that's why I suggest using **option 1**  https://github.com/gantry/gantry5/issues/3138#issuecomment-1495566494 if users already downloaded the package 5.5.16 
See 
![broken](https://user-images.githubusercontent.com/2299347/229741796-33fee415-ae6a-4f7e-8c1a-ecb4095d085a.png)

"I was replacing tokens around **2 AM** so probably missed something :("
Yes I see, but the commits effect to CI build only and that's why the CI build package working :)

**Comment by hexplor** [comment]
Ok, should be fine now. Version numbering and date fixed and re-released. Please re-check and let me know. Thank you for your help!

**Comment by OldMan-Jakob** [comment]
Still not working.
danger
"The checksum verification failed. Please make sure you are using the correct update server!"

**Comment by tonamphong** [comment]
> Ok, should be fine now. Version numbering and date fixed and re-released. Please re-check and let me know. Thank you for your help!

Just downloaded [joomla-pkg_gantry5_v5.5.16.zip](https://github.com/gantry/gantry5/releases/download/5.5.16/joomla-pkg_gantry5_v5.5.16.zip) here https://github.com/gantry/gantry5/releases/tag/5.5.16 and it's working.

@OldMan-Jakob Hexplor disabled the auto update via Joomla back-end for debugging, I think the update system will be revert back soon

**Comment by gphood** [comment]
I've just installed the updated package file onto a test site running a template based on Helium and that looks to be working properly as well.

**Comment by morphinestyle** [comment]
Same issue.

**Comment by tonamphong** [comment]
@hexplor 
Can you check the "intro" screen for template installation/update with the newest 5.5.16 ?
With 5.5.15, we will get the result like this (detailed information)

![intro-1](https://user-images.githubusercontent.com/2299347/230526346-f6d15e06-ac15-41de-a3eb-afdad58259a6.png)

However, if you use framework 5.5.16, the fresh template installation or update just appear the short message like this

![intro-2](https://user-images.githubusercontent.com/2299347/230526448-3f57dd4c-06e1-4452-b167-154863cbe0d3.png)

**Comment by fseesink** [comment]
Maybe this is why I see what I do on the Grav plugin pages.  Wrote the following last week in the gantry/gantry5 Matrix room:

Not quite sure whether to ask this here or over in the Grav Discord.  But thought I should mention it here since it is about Gantry itself.

I just realized that Gantry 5 released v5.5.16 a few weeks ago when I was looking at its GitHub repo.  I typically see update notifications within my Grav sites to let me know an update is available (whether for Grav, a plugin like Gantry 5, or a theme like Helium).  But I never got such a notification.

Checking the Grav website where users can search for plugins, I see Gantry 5 is still listed as 5.5.15 there.  This can be seen by searching for "Gantry" here:  https://getgrav.org/downloads/plugins
...
Just to add to this, searching the [getgrav.org](http://getgrav.org/) site for themes, they do show both the Hydrogen and Helium themes with updates just 2 weeks ago, but both say v5.5.15.  That seems odd, since Gantry v5.5.15 goes back to 5 Sep 2022.
Oh wait... I think I see it now.  Both the themes (Hydrogen, Helium) AND Gantry 5 are all listed as v5.5.15 on the [getgrav.org](http://getgrav.org/) site, NOT 5.5.16.
So likely a case of the version not being updated, which in turn means when my Grav sites check for updates, they don't see anything new.

**Comment by hexplor** [comment]
> Maybe this is why I see what I do on the Grav plugin pages. Wrote the following last week in the gantry/gantry5 Matrix room:
> 
> Not quite sure whether to ask this here or over in the Grav Discord. But thought I should mention it here since it is about Gantry itself.
> 
> I just realized that Gantry 5 released v5.5.16 a few weeks ago when I was looking at its GitHub repo. I typically see update notifications within my Grav sites to let me know an update is available (whether for Grav, a plugin like Gantry 5, or a theme like Helium). But I never got such a notification.
> 
> Checking the Grav website where users can search for plugins, I see Gantry 5 is still listed as 5.5.15 there. This can be seen by searching for "Gantry" here: https://getgrav.org/downloads/plugins ... Just to add to this, searching the [getgrav.org](http://getgrav.org/) site for themes, they do show both the Hydrogen and Helium themes with updates just 2 weeks ago, but both say v5.5.15. That seems odd, since Gantry v5.5.15 goes back to 5 Sep 2022. Oh wait... I think I see it now. Both the themes (Hydrogen, Helium) AND Gantry 5 are all listed as v5.5.15 on the [getgrav.org](http://getgrav.org/) site, NOT 5.5.16. So likely a case of the version not being updated, which in turn means when my Grav sites check for updates, they don't see anything new.

We are indeed aware of the situation with Gantry 5.5.16. The release was slowly rolled out, but we had to halt it temporarily because some users experienced serious issues with this version. Our development team is currently working on addressing these issues and we'll likely release an updated version, probably Gantry 5.5.17, to fix these problems.

Once the updated version is ready and tested, we'll make sure to distribute it through the appropriate update channels.

**Comment by dagroupinc** [comment]
Are there any updates to this? We had our dev site working just fine and when we installed it (up one directory same server) we get the error:

Failed to load 'g5_helium' template: Please install Gantry 5 Framework!

**Comment by N8Solutions** [comment]
@dagroupinc Glad you got your problem fixed. Just wanted to point out that the latest version is now v5.5.17.

**Comment by dagroupinc** [comment]
Yes, that's what we're using. Thanks!
